dotCMS/src/main/webapp/html/portlet/ext/contentlet/edit_contentlet.jsp

```diff
@@ -555,17 +537,28 @@ dojo.addOnLoad(function () {
 <script type="text/javascript">
 	dojo.addOnLoad(function () {
 		dojo.style(dijit.byId('savingContentDialog').closeButtonNode, 'visibility', 'hidden');
-
-		var tab = dijit.byId("mainTabContainer");
- 		dojo.connect(tab, 'selectChild',
- 				function (evt) {
- 				 	selectedTab = tab.selectedChildWidget;
-
-					var isCustomTab = false;
-					if (selectedTab.class != undefined && selectedTab.class == "custom-tab") {
-						isCustomTab = true;
-					}
- 				});
+		
+        var tab = dijit.byId("mainTabContainer");
+ 		dojo.connect(tab, 'selectChild', function (evt) {
+            selectedTab = tab.selectedChildWidget;
+
+            var isCustomTab = false;
+            if (selectedTab.class != undefined && selectedTab.class == "custom-tab") {
+                isCustomTab = true;
+            }
+        });
+
+        var customEvent = document.createEvent("CustomEvent");
+        customEvent.initCustomEvent("ng-event", false, false,  {
+            name: "edit-contentlet-loaded",
+            data: {
+                contentType: '<%=CacheLocator.getContentTypeCache().getStructureByInode(structure.getInode() ).getName()%>'
+            }
+        });
+        setTimeout(function() {
```

Comment:
This timeout is because in the angular side, we're binding the custom events when the page load and without the timeout this events is triggered before is getting binded at the nd side.